helm/delete-all.sh

```diff
@@ -13,10 +13,14 @@ function Deleting_All() {
 
   helm -n $SOFTHSM_NS delete softhsm-esignet
 
-  declare -a module=("redis"
-                     "esignet"
-         "oidc-ui"
-                     )
+  declare -a module=("postgres"
```

Comment:
correct the sequence. It should be opposite of the installation sequence.

Reply:
done